functions/v2/cloudevent/log-cloudevent/pom.xml

```diff
@@ -16,9 +16,7 @@
   limitations under the License.
 -->
 
```

Comment:
Did you mean to change this file?

I think we should put this sample in its own directory, e.g. `functions/v2/cloudevent/label-compute-instance`

Reply:
We probably want to restructure this directory without so much nesting. Currently it would be:
```
- helloworld/
  - hello-gcs/
- pubsub/
  - subscribe-to-topic/
- cloudevents/
  - log-cloudevent/
  - autolabelinstance/ (if moved into another project)
```
But we may just want to use the top level folder with individual files for functions
```
- helloworld/
  - hello-gcs.java
- pubsub/
  - subscribe-to-topic.java
- cloudevents/
  - log-cloudevent.java
  - autolabelinstance.java
```

Thoughts?

Reply:
I'd be happy with `functions/v2/label-compute-instance`, if you want to remove the `cloudevent` part.